docs/docs/documentation/getting-started/installation/postgres.md

```diff
@@ -5,40 +5,41 @@ PostgreSQL might be considered if you need to support many concurrent users. In
 **For Environment Variable Configuration, see** [Backend Configuration](./backend-config.md)
 
 ```yaml
----
```

Comment:
Is this an accident?

Reply:
No. It isn't required for valid yaml and most example compose files do not use it. Does it break something by removing it?

Reply:
FWIW, I don't care one way or the other about the `---` syntax, but I asked the question because the other sample file didn't have the change. I like to think _including_ `---` is _technically correct_, but I'd never actually write it myself.
No concerns, now that you've removed it from the other file too.

Reply:
Yeah, I didn't realize this until much later, sorry. I made sure to correct it when I noticed. :)